pkg/devcontainers/features/local.go

```
@@ -0,0 +1,101 @@
+		}
+
+		return copyFile(path, target)
+	})
+}
+
+func copyFile(src, dst string) error {
+	in, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer in.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `in.Close` is not checked

(errcheck)